### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by prog893**
- Project name description
- awesome-like modules list
- Logos and license information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+    - [lambda-kinesis-to-es](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to_es) - Kinesis Data Streams -> Elasticsearch transfer
+    - [lambda-es-cleaner](https://github.com/baikonur-oss/terraform-aws-lambda-es_cleaner) - Old indices cleaner for Elasticsearch Service
+    - [lambda-kinesis-forward](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-forward) - Kinesis Data Streams -> Kinesis Data Streams forwarder/router module
+
+- GCP
+  - coming soon
+
+
+## Contributing
+PRs, issues, and any other form of feedback is very welcome in all repositories under baikonur-oss organization! :smile:
```
Comment (toVersus) [inline README.md:44]:
```suggestion
PRs, issues, and any other form of feedback are very welcome in all repositories under baikonur-oss organization! :smile:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+  <br>
+  <br>
+</p>
+
+
+# Baikonur Project Homepage
+
+Welcome to Baikonur!
+
+Baikonur is an open source [Terraform Modules](https://www.terraform.io/docs/modules/usage.html) project by [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/)
```
Comment (toVersus) [inline README.md:13]:
```suggestion
Baikonur is an open source [Terraform Modules](https://www.terraform.io/docs/modules/usage.html) project owned by [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) 
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+
+# Baikonur Project Homepage
+
+Welcome to Baikonur!
+
+Baikonur is an open source [Terraform Modules](https://www.terraform.io/docs/modules/usage.html) project by [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) 
+
+Baikonur started as an internal project in 2018, and quickly grew to over 50 modules.
+We are releasing select most popular and unique modules in this organization.
```
Comment (toVersus) [inline README.md:16]:
```suggestion
We are releasing carefully picked most popular and unique modules in this organization.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+
+Baikonur project name comes from [Baikonur Cosmodrome](https://en.wikipedia.org/wiki/Baikonur_Cosmodrome), 
+a spaceport located in Kazakhstan known for launching the first artificial satellite and the first human spaceflight.
+
+Modules published here are also released to Terraform Public Module Registry: [Baikonur top page](https://registry.terraform.io/modules/baikonur-oss).
```
Comment (toVersus) [inline README.md]:
```suggestion
Modules released here are also published to Terraform Public Module Registry: [Baikonur top page](https://registry.terraform.io/modules/baikonur-oss).
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,54 @@
...
+
+## Usage
+Follow instructions in repository READMEs or [Terraform Module Registry pages](https://registry.terraform.io/modules/baikonur-oss).
+Refer to examples in repositories, and don't forget to use proper [versioning](https://www.terraform.io/docs/configuration/modules.html#module-versions).
+
+## Modules
+
+- AWS
+  - [iam-nofile](https://github.com/baikonur-oss/terraform-aws-iam-nofile) - module to make creating AWS IAM Roles easier. You can write role policy document with inline(heredoc) syntax, so you do not have to use [template rendering](https://www.terraform.io/docs/providers/template/d/file.html) in order to use variables.
```
Comment (toVersus) [inline README.md]:
```suggestion
  - [iam-nofile](https://github.com/baikonur-oss/terraform-aws-iam-nofile) - module to make it easier to create AWS IAM Roles. You can write role policy document with inline(heredoc) syntax, so you do not have to use [template rendering](https://www.terraform.io/docs/providers/template/d/file.html) in order to use variables.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,54 @@
...
+    - [lambda-kinesis-forward](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-forward) - Kinesis Data Streams -> Kinesis Data Streams forwarder/router module
+
+- GCP
+  - coming soon
+
+
+## Contributing
+PRs, issues, and any other form of feedback is very welcome in all repositories under baikonur-oss organization! :smile:
+
+If you have a module you would like to propose to Baikonur, let us know in Issues of this repository.
```
Comment (toVersus) [inline README.md]:
```suggestion
If you have a module you would like to donate to Baikonur, let us know in Issues of this repository.
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,54 @@
...
+Follow instructions in repository READMEs or [Terraform Module Registry pages](https://registry.terraform.io/modules/baikonur-oss).
+Refer to examples in repositories, and don't forget to use proper [versioning](https://www.terraform.io/docs/configuration/modules.html#module-versions).
+
+## Modules
+
+- AWS
+  - [iam-nofile](https://github.com/baikonur-oss/terraform-aws-iam-nofile) - module to make creating AWS IAM Roles easier. You can write role policy document with inline(heredoc) syntax, so you do not have to use [template rendering](https://www.terraform.io/docs/providers/template/d/file.html) in order to use variables.
+
+  - Modules for logging with Kinesis and Lambda
+    - [lambda-kinesis-to-fluent](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to_fluent) - Kinesis Data Streams -> Fluent transfer
```
Comment (toVersus) [inline README.md]:
```suggestion
    - [lambda-kinesis-to-fluent](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to-fluent) - Kinesis Data Streams -> Fluent transfer
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,54 @@
...
+    - [lambda-kinesis-to-s3](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to_s3) - Kinesis Data Streams -> S3 transfer
```
Comment (toVersus) [inline README.md]:
```suggestion
    - [lambda-kinesis-to-s3](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to-s3) - Kinesis Data Streams -> S3 transfer
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,54 @@
...
+    - [lambda-kinesis-to-es](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to_es) - Kinesis Data Streams -> Elasticsearch transfer
```
Comment (toVersus) [inline README.md]:
```suggestion
    - [lambda-kinesis-to-es](https://github.com/baikonur-oss/terraform-aws-lambda-kinesis-to-es) - Kinesis Data Streams -> Elasticsearch transfer
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,54 @@
...
+    - [lambda-es-cleaner](https://github.com/baikonur-oss/terraform-aws-lambda-es_cleaner) - Old indices cleaner for Elasticsearch Service
```
Comment (toVersus) [inline README.md]:
```suggestion
    - [lambda-es-cleaner](https://github.com/baikonur-oss/terraform-aws-lambda-es-cleaner) - Old indices cleaner for Elasticsearch Service
```

**Review by toVersus — COMMENTED** [review]
I'm not confident but some suggestions...

File: README.md (hunk 1 above)
Comment (prog893) [reply README.md:44]: https://english.stackexchange.com/questions/2121/which-is-correct-feedback-is-welcome-or-feedback-is-welcomed

File: README.md (hunk 2 above)
Comment (prog893) [reply README.md:13]: Hmm, not exactly owned, but more like initiated and managed.

Difficult matter, should be discussed separately.

File: README.md (hunk 3 above)
Comment (prog893) [reply README.md:16]: https://www.dailywritingtips.com/select-vs-selected/

Comment (toVersus) [reply README.md:16]: ~~Sure. Go ahead as is.~~

File: README.md (hunk 2 above)
Comment (toVersus) [reply README.md:13]: Sure. Go ahead as is.

**Comment by prog893** [comment]
Comments resolved, can I get approved?

**Review by toVersus — APPROVED** [review]
LGTM

**Pull request merged**